main/src/test/java/mockit/MisusedFakingAPITest.java

```diff
@@ -95,11 +95,12 @@ public void applyTheSameFakeForAClassTwice() {
      */
     @Test
     public void fakeAPrivateMethod() {
-        thrown.expect(IllegalArgumentException.class);
-        thrown.expectMessage("Unsupported fake for private method");
-        thrown.expectMessage("Component");
-        thrown.expectMessage("checkCoalescing()");
-        thrown.expectMessage("found");
+        // Enabled to fake a private constructor.
```

Comment:
Same on these, just a bit more clarlity as to why we are keeping commented out code.